notebooks/evaluation/rag_experiment_evaluation.ipynb

```diff
@@ -0,0 +1,2086 @@
+      {
+         "cell_type": "code",
+         "execution_count": 6,
+         "metadata": {},
+         "outputs": [],
+         "source": [
+            "from pathlib import Path\n",
+            "\n",
+            "ROOT = Path.cwd().parents[1]\n",
+            "EVALUATION_DIR = ROOT / \"notebooks/evaluation\"\n",
+            "\n",
+            "V_ROOT = EVALUATION_DIR / f\"data/{DATA_VERSION}\"\n",
+            "V_RAW = V_ROOT / \"raw\"\n",
+            "V_SYNTHETIC = V_ROOT / \"synthetic\"\n",
+            "V_CHUNKS = V_ROOT / \"chunks\"\n",
+            "V_RESULTS = V_ROOT / \"results\"\n",
+            "\n",
+            "V_ROOT.mkdir(parents=True, exist_ok=True)\n",
+            "V_RAW.mkdir(parents=True, exist_ok=True)\n",
+            "V_SYNTHETIC.mkdir(parents=True, exist_ok=True)\n",
+            "V_CHUNKS.mkdir(parents=True, exist_ok=True)\n",
+            "V_RESULTS.mkdir(parents=True, exist_ok=True)"
+         ]
+      },
+      {
+         "cell_type": "markdown",
+         "metadata": {},
+         "source": [
+            "To save on API costs, we only need to generate a particular version of the evaluation dataset once. If you are using a previously generaterated evalutation dataset, **please download it from shared team location (Google Drive).**"
+         ]
+      },
+      {
+         "cell_type": "markdown",
+         "metadata": {},
+         "source": [
+            "---------"
+         ]
+      },
+      {
+         "cell_type": "markdown",
+         "metadata": {},
+         "source": [
+            "#### Imports <a id=\"imports\"></a>"
+         ]
+      },
+      {
+         "cell_type": "code",
+         "execution_count": 7,
+         "metadata": {},
+         "outputs": [],
+         "source": [
+            "# Add autoreloatd\n",
+            "%reload_ext autoreload\n",
+            "%autoreload 2"
+         ]
+      },
+      {
+         "cell_type": "code",
+         "execution_count": null,
+         "metadata": {},
+         "outputs": [],
+         "source": [
+            "from jose import jwt\n",
+            "from uuid import UUID\n",
+            "import requests\n",
+            "import json\n",
+            "import pandas as pd\n",
+            "import pickle\n",
+            "from dataclasses import asdict\n",
+            "from IPython.display import clear_output"
+         ]
+      },
+      {
+         "cell_type": "code",
+         "execution_count": 9,
+         "metadata": {},
+         "outputs": [],
+         "source": [
+            "from dotenv import find_dotenv, load_dotenv\n",
+            "_ = load_dotenv(find_dotenv())"
+         ]
+      },
+      {
+         "cell_type": "markdown",
+         "metadata": {},
+         "source": [
+            "[Back to top](#title)"
+         ]
+      },
+      {
+         "cell_type": "markdown",
+         "metadata": {},
+         "source": [
+            "-----------------"
+         ]
+      },
+      {
+         "cell_type": "markdown",
```

Comment:
Could we maybe signpost where people could skip ahead if they've already run the baselining notebook?

Reply:
Start Redbox Running locally